### ISSUE TITLE
[Transforms] Comments on accessors should follow their semicolons/commas. 

### ISSUE THREAD

**Issue by mhegazy** [Microsoft/TypeScript#7893]
**Expected**:

```js
 var DbSet = (function () {
        function DbSet() {
        }
        Object.defineProperty(DbSet.prototype, "entityType", {
            get: function () { return this._entityType; } // used to ICE without return type annotation
            ,
            enumerable: true,
            configurable: true
        });
        return DbSet;
    }());
```

**Actual**:

```js
var DbSet = (function () {
        function DbSet() {
        }
        Object.defineProperty(DbSet.prototype, "entityType", {
            get: function () { return this._entityType; },
            enumerable: true,
            configurable: true
        }) // used to ICE without return type annotation
        ;
        return DbSet;
```

Failed tests:
*  tests/cases/compiler/genericReturnTypeFromGetter1.ts 
* tests\cases\compiler\staticVisibility.ts

**Comment by mhegazy** [comment]
This is probably fixed now.

**Comment by mhegazy** [comment]
For tests\cases\compiler\superCallInNonStaticMethod.ts

Expected: 
```js
    Object.defineProperty(Other.prototype, "accessor", {
        // in a getter
        get: function () {
            _super.prototype.instanceMethod.call(this);
            return 0;
        },
        // in a setter
        set: function (value) {
            _super.prototype.instanceMethod.call(this);
        },
        enumerable: true,
        configurable: true
    });
```

Actual:
```js
// in a getter
    Object.defineProperty(Other.prototype, "accessor", {
        get: function () {
            _super.prototype.instanceMethod.call(this);
            return 0;
        },
        set: 
        // in a setter
        function (value) {
            _super.prototype.instanceMethod.call(this);
        },
        enumerable: true,
        configurable: true
    });
```

**Comment by rbuckton** [comment]
Fixed in #7844.
